Package.swift

```diff
@@ -21,6 +21,23 @@ let package = Package(
             name: "Wire",
             path: "wire-library/wire-runtime-swift/src/main/swift"
         ),
+        // The tests depend on valid protos via gradle
+        // ./gradlew -p wire-library :wire-runtime-swift:generateTestProtos
+        .testTarget(
```

Comment:
By defining these test targets, most swift development can be done seamlessly in Xcode, including test validation